src/app/public/modules/inline-delete/inline-delete.component.html

```diff
@@ -0,0 +1,41 @@
+<div
+  class="sky-inline-delete sky-inline-delete-{{type}}"
+  role="alertdialog"
+  [attr.aria-describedby]="assistiveTextId"
+  [attr.aria-label]="'skyux_inline_delete_confirm_deletion' | skyLibResources"
+>
+  <span
+   class="sky-inline-delete-assistive-text"
+   [id]="assistiveTextId"
+  >
+    {{ 'skyux_inline_delete_assistive_text' | skyLibResources }}
+  </span>
+  <div *ngIf="pending"
+    class="sky-inline-delete-wait"
+  >
+    <sky-wait
+      [isWaiting]="true"
+    ></sky-wait>
+  </div>
+  <div
+    class="sky-inline-delete-content"
+    [@skyAnimationEmerge]="animationState"
```

Comment:
I noticed this zooms in, instead of sliding in - as the prototype shows. Was this deliberate?

Reply:
I believe this was a change via @Blackbaud-ToddRoberts but could be remember wrong. I remember there being some conversation around the animation in the design needing to change due to the motion guidelines changing between design and implementation.

Reply:
Heard back from Todd that the animation is good to go now :)